com.unity.multiplayer.mlapi/Runtime/Serialization/BitWriter.cs

```diff
@@ -870,12 +870,12 @@ public void WriteByteArray(byte[] b, long count = -1)
         /// </summary>
         /// <param name="buffer"></param>
         /// <param name="targetSize"></param>
-        public void WriteBytes(byte[] buffer, long targetSize)
+        public void WriteBytes(byte[] buffer, long targetSize, int offset = 0)
         {
 
             long TargetSize = targetSize;
-            long LargeInt64Blocks = (long)(TargetSize * 0.125f);
-            int IndexOffset = 0;
+            long LargeInt64Blocks = TargetSize >> 3;
```

Comment:
Please add a comment :-)
e.g.: divides by 8, which is the number of bytes in a 64bit field, or something

Reply:
Done